lib/peep.rb

```diff
@@ -0,0 +1,26 @@
+
+require 'pg'
+
+class Peep
+
+  def self.all
+    if ENV['ENVIRONMENT'] = 'test'
```

Comment:
you need a double '==' here!

Reply:
this is **setting** `ENV['ENVIRONMENT']` to `"test"` - that's why your app is connecting to the wrong database.

Reply:
you're an angel, that was ruining my day